yogesh/tests/acceptance/PageObject/entityPage.js

```diff
@@ -0,0 +1,37 @@
+class Entity{
+    constructor(){
+        this.dialogInputSelector = '//div[@class="card-content"]/input[@type="text"]';
+        this.lastNavaigatedFolderSelector = '//div[@class="breadcrumbs"]/span[last()]/a'
+        this.contentEditorSelector = '//form[@id="editor"]/textarea[@class="ace_text-input"]'
+        this.editorContent = '//div[@class="ace_content" ]/div[@class="ace_layer ace_text-layer"]/div[@class="ace_line_group"]/div[@class="ace_line"]'
+        this.createButtonSelector =this.getHomePageButtonSelector('Create')
+    }
+    getHomePageButtonSelector(title){
+        return `//button[@title="${title}"]`
+    }
+
+    getFileSelector(filename){
+        return `//div[@role="button" and  @aria-label="${filename}"]`
+    }   
```

Comment:
If  you're trying to make your selector dynamic, there's a NodeApi called `util.format` . That would be easy than calling a function inside constructor. 
AFAIK, functions aren't generally defined inside constructor.

Reply:
sure